src/features/people/PeopleDetails/index.js

```diff
@@ -18,20 +25,94 @@ const PersonDetails = () => {
 
     const details = useSelector(selectDetails);
     const status = useSelector(selectStatus);
+    const actorCast = useSelector(selectPersonCast);
+    const actorCrew = useSelector(selectPersonCrew);
+    const [genres, setGenres] = useState([]);
+
+    useEffect(() => {
+        const fetchGenres = async () => {
+            const genresData = await getGenreList();
+            setGenres(genresData.genres);
+        };
+        fetchGenres();
+    }, []);
+
+    const nameGenres = (genreIds) => {
+        if (!genres || genres.length === 0) {
+            return [];
+        }
+
+        return genreIds.map((id) => {
+            const genre = genres.find((genre) => genre.id === id);
+            return genre ? genre.name : '';
+        });
+    };
+
+    const getReleaseYear = (releaseDate) => {
+        const date = new Date(releaseDate);
+        return date.getFullYear();
+    };
+
+    const formatVote = (vote) => vote.toLocaleString(
+        'pl-PL',
+        {
+            maximumFractionDigits: 1,
+        }
+    );
 
     return (
         status === "loading" ? <div>Loading...</div>
             : status === "error" ? <div>Error!</div>
                 : details ? (
-                    <>
-                        <PersonDetailsTile
-                            poster={details.profile_path}
+                    <Main>
+                        <DetailsTile
+                            posterPath={details.profile_path}
                             title={details.name}
-                            overview={details.biography}
-                            dateOfBirth={details.birthday}
-                            placeOfBirth={details.place_of_birth}
+                            description={details.biography}
+                            firstData={details.birthday}
+                            secondData={details.place_of_birth}
                         />
-                    </>
+
+                        <Section>
+                            <SectionTitle>Movies - cast ({actorCast.length})</SectionTitle>
+                            <SectionWrapper>
```

Comment:
I've changed SectionWrapper to, in this case, LargeListWrapper so import form movieDetails/styled is no longer necessary 💪🏻

Reply:
Got it! Thanks for the optimization. I'll update the component accordingly and remove the import from movieDetails/styled. Appreciate the suggestion! 🚀👍🏻